### ISSUE TITLE
Update needed:urn:uuid:1d133e63-3ab1-47b4-9fc9-e883b68bc7f3

### ISSUE THREAD

**Issue by kevinlove** [iDigBio/idb-us-collections#1804]
https://www.idigbio.org/portal/collections/1d133e63-3ab1-47b4-9fc9-e883b68bc7f3

**Comment by ShariEllis** [comment]
from IH:
[1] 518 388 6243
Fax: [1] 518 388 6429 
It is housed in the Biology Department so maybe contact dept. chair
Kathleen LoGiudice, Department Chair,  biologychair@union.edu

**Comment by ShariEllis** [comment]
Staff directory doesn't list anyone in charge of the herbarium

**Comment by kevinlove** [comment]
@ShariEllis , I found this information via IH:

http://sweetgum.nybg.org/science/ih/person_details.php?irn=135599

Are you checking the IH records before searching for other contact info? I'd like to defer to IH before reaching out to these institutions.

**Comment by ShariEllis** [comment]
hmmm...I must not have? I have been checking IH first and if the name is different than the one we had, I put it in the comment. There was a case yesterday though (a govt collection) where we had the same person as IH and I couldn't spot anything wrong with the email address. So I googled the agency and found a new name/director/curator. So IH wasn't up-to-date on that one because it obviously bounced. Sorry for the error on this one! Guess I got too into googling...
